### PR TITLE
update readme to discuss schema updates and flyway migration.

### PR DESCRIPTION
**Pull request by ShovelHand**
Based on  what you showed me during our call today.

### CODE REVIEW

File: openshift/README.md [hunk 1]
```diff
@@ -38,3 +38,12 @@ This time we do not build images, instead, we tag the image that we used for _te
...
 again, your changes now are all in the Prod environment!
+
+###### Verify Release
+
+Once you've finsihed your release to prod, check that it was successful.
+If your release included changes to the database schema, verify that those changes are reflected in the patroni instance on OpenShift. From https://console.apps.silver.devops.gov.bc.ca/topology/ns/platform-registry-prod, click on the registry-api pod, and access its terminal. Change the "Connecting to" option at the top of the terminal window from "registry-api" to "flyway-migration". The command ```flyway info``` will show the history and status of each migration that has taken place. The command may take a few moments. Your most recent one should be at the bottom of the list, with its corrent version number, and a status of "Success".
+
+If your release does not appear, or its status is not good, see if the command ```flyway-migrate``` fixes things.
```
Comment (w8896699) [inline openshift/README.md:47]: two separate words.

**Review by w8896699 — CHANGES_REQUESTED** [review]

File: openshift/README.md (hunk 1 above)
Comment (ShovelHand) [reply openshift/README.md:47]: fixed two spelling mistakes. "finsihed" and "corrent".

Comment (w8896699) [reply openshift/README.md:47]: cmd is `flyway migrate`

**Review by w8896699 — CHANGES_REQUESTED** [review]
almost there

**Pull request merged**